pyproject.toml

```
@@ -25,6 +25,7 @@ dependencies = [
     "healpy>=1.15.0",
     "numpy>=2.1.1",
     "transformcl>=2022.8.9",
+    "typing_extensions",
```

Just wonder whether we want to pin this for consistency? Not fixed on the specific version
```suggestion
    "typing_extensions>=4.13.2",
```